src/main/java/org/owasp/validator/html/Policy.java

```diff
@@ -371,20 +371,7 @@ protected static Element getTopLevelElement(InputSource source, Callable<InputSo
         Exception thrownException = null;
         try {
             return getDocumentElementFromSource(source, validateSchema);
-        } catch (SAXException e) {
-            thrownException = e;
-            if (!validateSchema) {
-                try {
-                    source = getResetSource.call();
-                    Element theElement = getDocumentElementFromSource(source, false);
-                    // We warn when the policy has an invalid schema, but schema validation is disabled.
-                    logger.warn("Invalid AntiSamy policy file: " + e.getMessage());
-                    return theElement;
-                } catch (Exception e2) {
-                    throw new PolicyException(e2);
-                }
-            } else throw new PolicyException(e);
-        } catch (ParserConfigurationException | IOException e) {
+        } catch (SAXException | ParserConfigurationException | IOException e) {
```

Comment:
Why was this code deleted? Was it moved somewhere else? If just deleted, I don't think that is desirable.

Reply:
As we discuss with @jeremiahjstacey , that was dead code. 
I can redo it, but it's uneachable

Reply:
It's truly unreachable, deleting it is OK. But when I test, I want to see warnings for each disabled situation for every type of Policy we can instantiate, including the new one causing all this extra work to be done.